workspaces/lightspeed/plugins/lightspeed/src/components/LightspeedPage.tsx

```diff
@@ -72,19 +75,31 @@ const LightspeedPageInner = () => {
     if (modelsItems.length > 0) setSelectedModel(modelsItems[0].value);
   }, [modelsItems]);
 
+  if (loading) {
+    return null;
+  }
   return (
     <Page themeId="tool">
       <Content className={classes.container}>
-        <LightspeedChat
-          selectedModel={selectedModel}
-          handleSelectedModel={item => {
-            setSelectedModel(item);
-          }}
-          models={modelsItems}
-          userName={profile?.displayName}
-          avatar={profile?.picture}
-          profileLoading={profileLoading}
-        />
+        {!hasViewAccess ? (
+          <Alert severity="warning" data-testid="no-permission-alert">
```

Comment:
Currently, most of the rhdh plugins with permission support uses this Alert as a common convention to surface the permission related warnings. we used this in tekton, argocd and quay as well.

Reply:
Okay, we can use the alert for now. But I think we should look at introducing a better error state in future. @ShiranHi WDYT?

Reply:
I agree that it doesn't look great, and yes, this is the default behavior provided by Backstage. 
I had created [a design](https://www.figma.com/design/GeVb7rc390TkxeQNnJIcBD/RHDH-Theme-(pages)?node-id=110-933&t=T1ZzXLYQ2UrCZrvY-1) for this scenario, but we didn’t get the chance to implement it as it was deprioritized. 
For now we can stick with this behavior unless you have the capacity to implement the new design, in which case we’ll also incorporate it into our plugins.

Reply:
Agreed, lets stick with this for now and will revisit to enhance the error states.

Reply:
@rohitkrai03 @ShiranHi @aprilma419  I have looked into the backstage error state component and I was able to modify it a bit to best match the error state design outlined in the above figma file.

For the CTA button, I've used `Read more` text and on clicking, it will redirect to the [permissions docs](https://github.com/redhat-developer/rhdh-plugins/blob/3b33f63301fe9d392816a6fc99d2c93a262a8d16/workspaces/lightspeed/plugins/lightspeed/README.md#permission-framework-support) README file  (added in this PR) of the lightspeed plugin. 
![Missing permissions](https://github.com/user-attachments/assets/66ebbbcf-3150-4a4f-9c87-9e681416cc1b)

Reply:
Looks great! Thank you!
A minor improvement: when opening a new tab, it’s helpful to add an icon to the button (label then [this icon](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:open_in_new:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=open&icon.size=24&icon.color=%235f6368)) to clearly inform users about the action.
Also, if possible, adjusting the title to match the font weight and size used in the design would be ideal.

Reply:
Unfortunately backstage ErrorState component does not allow us to modify the font styles in the title as it accepts only string at the moment. 


I have added external link icon.
![image](https://github.com/user-attachments/assets/6f3a5b4a-409a-4cee-bbdb-dbb05b76af15)